### ISSUE TITLE
508-defect-3 [SCREENREADER]: International support information should programmatically describe the number field

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#23162]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user who relies on the `tab` key when completing forms, I expect non interactive content to be announced to me when I `tab` into the field it describes.

## Details

International support information may be missed by screen reader users who rely more on `tab` to complete forms. Upon beginning to edit the phone number field, should they press `tab` immediately, they will skip over the information.

## Acceptance Criteria

- [ ] International support information programmatically describes the number field using `aria-describedby`

## Steps to Recreate

My profile, user 36 Wesley, when editing any phone fields.

## Proposed Solution (if known)

Attach `aria-describedby` to the `<input>`

## Screenshots or Trace Logs

<img width="1121" alt="Screen Shot 2021-04-13 at 3 42 25 PM" src="https://user-images.githubusercontent.com/14154792/114611462-13a7d080-9c6f-11eb-8c20-ba93a7ac534a.png">

**Comment by ajakabcin** [comment]
@tmitchellgcio also moving this to next sprint to maintain focus on radio button work

**Comment by andaleliz** [comment]
FYI this should be a non-issue with the changes we implemented in 22533. I recommend we close this.

**Comment by joshkimux** [comment]
Got it! Closing 👍
